hack/cluster-sync.sh

```diff
@@ -40,8 +40,8 @@ function main() {
         exit 1
     fi
 
-    ./hack/deploy-to-nodes.sh
     ./hack/cluster-deploy.sh
+    ./hack/deploy-to-nodes.sh
```

Comment:
Does this change affect other things?

Reply:
no, and it is just because we need the NS created first